CONTRIBUTOR-V1.md

```diff
@@ -33,6 +33,7 @@ Sorting applied on the name is an alphabetical one.
 - Jim Manico - [jim@owasp.org](mailto:jim@owasp.org)
 - Manuel Aude Morales
 - Pawel Krawczyk
+- Pankaj Taneja
```

Comment:
This is for V1 cheatsheets - old cheatsheets hosted on OWASP wiki. Contributors for V2 (current) version are in GitHub. Please remove this line.